Sources/ArcGISToolkit/Components/Offline/PreplannedListItemView.swift

```diff
@@ -85,6 +88,7 @@ struct PreplannedListItemView: View {
            !isSelected {
             Button("Delete") {
                 model.removeDownloadedPreplannedMapArea()
+                onDeletion()
             }
             .tint(.red)
```

Comment:
There is no difference that I can see.

```suggestion
            Button("Delete", role: .destructive) {
                model.removeDownloadedPreplannedMapArea()
                onDeletion()

```

Reply:
This suggestion has the side effect of removing the list item without changing the list's data source. The destructive button style works well for the offline case since the list data source does not need to be reloaded and the `onDeletion` closure can be removed. However, the destructive button style does not work for the online case where a deleted list item should not be removed from the list.

Reply:
In that case, consider using a different button color and title ("Remove Cache"). Having it be so similar to a destructive delete button but not function like one is rather surprising.

Reply:
Okay, to support that the `PreplannedListItemView` would need a parameter to indicate that the view is online or offline to determine if the "remove cache" or "delete" button is used respectively.

Reply:
I will log an issue and address this in another PR.